### ISSUE TITLE
util.replaceecall is not a function

### ISSUE THREAD

**Issue by freak4pc** [neumino/reqlite#41]
I'm not sure exactly why this happens but running the following login query brings up this error 

```javascript
User.get(req.params.uid).run().then(u => {
                u.merge({lastLogin: new Date()}).save().then(result => {
                    req.session.loggedIn = {"uid": req.params.uid, "loginDate": u.lastLogin, "nickname": u.nickname}
                    res.sendOK("Welcome, " + u.nickname)
                    res.log(u.nickname.magenta + " just logged in " + "successfully".green)
                }).error(err => {
                    res.logError(err)
                    res.sendServerError(err.message)
                })
            }).catch(thinky.Errors.DocumentNotFound, err => {
                res.sendNotFound("The requested user does not exist")
                return
            }).error(err => {
                res.logError(err)
                res.sendServerError(error)
            })
```

Here's the full throw: 

```
Possibly unhandled Error: util.replaceecall is not a function.
    at model.Document._onSaved (/Users/freak4pc/Work/DreamDate/DreamDate-API/node_modules/thinky/lib/document.js:813:12)
    at /Users/freak4pc/Work/DreamDate/DreamDate-API/node_modules/thinky/lib/document.js:782:16
    at tryCatch1 (/Users/freak4pc/Work/DreamDate/DreamDate-API/node_modules/thinky/node_modules/bluebird/js/main/util.js:43:21)
    at Promise$_callHandler [as _callHandler] (/Users/freak4pc/Work/DreamDate/DreamDate-API/node_modules/thinky/node_modules/bluebird/js/main/promise.js:627:13)
    at Promise$_settlePromiseFromHandler [as _settlePromiseFromHandler] (/Users/freak4pc/Work/DreamDate/DreamDate-API/node_modules/thinky/node_modules/bluebird/js/main/promise.js:641:18)
    at Promise$_settlePromiseAt [as _settlePromiseAt] (/Users/freak4pc/Work/DreamDate/DreamDate-API/node_modules/thinky/node_modules/bluebird/js/main/promise.js:800:14)
    at Promise$_settlePromises [as _settlePromises] (/Users/freak4pc/Work/DreamDate/DreamDate-API/node_modules/thinky/node_modules/bluebird/js/main/promise.js:934:14)
    at Async$_consumeFunctionBuffer [as _consumeFunctionBuffer] (/Users/freak4pc/Work/DreamDate/DreamDate-API/node_modules/thinky/node_modules/bluebird/js/main/async.js:75:12)
    at Async$consumeFunctionBuffer (/Users/freak4pc/Work/DreamDate/DreamDate-API/node_modules/thinky/node_modules/bluebird/js/main/async.js:38:14)
    at doNTCallback0 (node.js:408:9)
```

**Comment by freak4pc** [comment]
From my 2 minutes of debugging the throw happens inside the merge/save block e.g.

```javascript
u.merge({lastLogin: new Date()}).save().then(result => {
                    req.session.loggedIn = {"uid": req.params.uid, "loginDate": u.lastLogin, "nickname": u.nickname}
                    res.sendOK("Welcome, " + u.nickname)
                    res.log(u.nickname.magenta + " just logged in " + "successfully".green)
                }).error(err => {
                    res.logError(err)
                    res.sendServerError(err.message)
                })
```

**Comment by neumino** [comment]
Looks like my previous message wasn't sent.
It should be fixed in 2.0.4.

**Comment by freak4pc** [comment]
Perfect. thanks :)
